static/src/javascripts/projects/common/modules/onward/popular.js

```diff
@@ -44,7 +46,11 @@ define([
     };
 
     MostPopular.prototype.prerender = function () {
-        if (!(config.page.shouldHideAdverts || this.mobileMaximumSlotsReached())) {
+        if (!(
+            config.page.shouldHideAdverts || /* sensitive pages */
```

Comment:
Can we not get a lower level fix for this or move that logic up into the `shouldHideAdverts` definition, rather than having to check for this section all over the place?